scripts/publish.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+source $OKTA_HOME/$REPO/scripts/setup.sh
+
+cd $OKTA_HOME/$REPO
+
+PUBLISH_SHA="$(git rev-parse --short $SHA)"
+PUBLISH_REGISTRY="${ARTIFACTORY_URL}/api/npm/npm-topic"
+CURRENT_VERSION=$(< lerna.json jq -r '.version')
+TAGGED_VERSION=$CURRENT_VERSION-$PUBLISH_SHA
+
+npm config set @okta:registry ${PUBLISH_REGISTRY}
+
+function lerna_publish() {
+  MY_CMD="yarn run lerna publish from-package --force-publish=* --ignore-changes --no-push --no-git-tag-version --no-verify-access --registry \"${PUBLISH_REGISTRY}\" --yes"
+  echo "Running ${MY_CMD}"
+  ${MY_CMD}
+}
+
+# update version with commit SHA to allow lerna to publish
+FILES_TO_UPDATE_VERSION="lerna.json packages/odyssey-design-tokens/package.json packages/odyssey-babel-preset/package.json packages/odyssey-babel-loader/package.json packages/odyssey-react-mui/package.json packages/browserslist-config-odyssey/package.json"
+for PATH_AND_FILE in $FILES_TO_UPDATE_VERSION; do
+  FULL_PATH="$OKTA_HOME/$REPO/$PATH_AND_FILE"
+  json_contents="$(jq '.version = "'$TAGGED_VERSION'"' $FULL_PATH)" && \
+  echo -E "${json_contents}" > $FULL_PATH
+  git update-index --assume-unchanged $FULL_PATH
+done
+
+echo "Publishing to artifactory"
+# mark files as unchanged so lerna can publish commit
+git update-index --assume-unchanged scripts/publish.sh
```

Comment:
Why's the publish script showing it's changed?

Reply:
This one I am sort of stumped on, I am not sure what lerna is seeing as being changed . I have the --ignore-changes flag passed to lerna publish but it seems to ignore this flag (I believe they designed it this way unfortunately). Should we look more into this?

Reply:
Since this isn't something associated with source code, we can take this as an action item to research later, it has to be somewhere between lerna and bacon.

Reply:
I figured out what needed to be done (we do this in the SIW) adding `git checkout .` prevents local changes. I tested with this and it publishes successfully.